README.md

```diff
@@ -1,44 +1,46 @@
 # Jamulus Website
 
-This is the home of the [Jamulus website](https://jamulus.io) and wiki.
-
-## Tech
-
-Built with Jekyll and Fox-CSS
+This is the home of the [Jamulus website](https://jamulus.io). The goal of this page is to explain what Jamulus is and how to use it.
 
 ## Contribute and translation
```

Comment:
"Contribute **a** translation"?

Reply:
Don't give up! These are in total 3 files. The file you saw here is the README file which links to the contributing file.

Reply:
Yes,
> Contribute a translation

is better since we talk about translation afterwards.